controller/internal/enforcer/applicationproxy/http/http.go

```diff
@@ -386,7 +390,8 @@ func (p *Config) processNetRequest(w http.ResponseWriter, r *http.Request) {
 	if noNetAccessPolicy == nil && networkPolicy.Action.Rejected() {
 		http.Error(w, fmt.Sprintf("Access denied by network policy"), http.StatusNetworkAuthenticationRequired)
 		record.Source.Type = collector.EndPointTypeExteranlIPAddress
-		record.Source.ID = collector.DefaultEndPoint
+		record.Source.ID = networkPolicy.ServiceID
+		record.PolicyID = networkPolicy.PolicyID
 		return
```

Comment:
Same as above. I am realizing that if you move it above the if statement it will populate the accepts as well.